inst/examples/chat.R

```diff
@@ -0,0 +1,79 @@
+library(shiny)
+library(purrr)
+library(dplyr)
+library(magrittr)
+
+ui <- shinyUI(fluidPage(
+  tags$head(
+    tags$style(HTML("
+                    #chatbox {
+                     padding: .5em;
+                     border: 1px solid #777;
+                     height: 300px;
+                     overflow-y: scroll;
+                    }
+                    "))
+  ),
+  #' maybe possible easier ?
+  tags$script("
```

Comment:
Please extract to the standalone js file. Here attach script through URL. (Static files are stored in `www` directory)